applications/app/controllers/HTTPSCookieController.scala

```diff
@@ -0,0 +1,22 @@
+package controllers
+
+import model.Cached
+import play.api.mvc.{DiscardingCookie, Cookie, Action, Controller}
+import scala.concurrent.duration._
+
+object HTTPSCookieController extends Controller {
+  private val HTTPSOptIn = "https_opt_in"
+  private val lifetime = 90.days.toSeconds.toInt
+
+  def optIn() = Action { implicit request =>
+    Cached(60)(SeeOther("/").withCookies(
```

Comment:
not worth worrying about due to the traffic levels, but you could have cached it an hour+ without worrying